README.md

```diff
@@ -1,126 +1,241 @@
+```
 
-Visit the [release page](https://github.com/AntelopeIO/leap/releases) for Ubuntu binaries. This is the fastest way to get started with the software.
+<details> <summary>Ubuntu derivatives</summary>
 
-### Building From Source
+We currently only support Ubuntu but, if you want to try running our software on an Ubuntu derivative such as Linux Mint, you can determine the version of Ubuntu it was built from using the following command.
+```bash
+cat /etc/upstream-release/lsb-release
+```
 
-Recent Ubuntu LTS releases are the only Linux distributions that we fully support. Other Linux distros and other POSIX operating systems (such as macOS) are tended to on a best-effort basis and may not be full featured. Notable requirements to build are:
-* C++17 compiler and standard library
-* boost 1.67+
-* CMake 3.8+
-* (for Linux only) LLVM 7 - 11 (newer versions do not work)
+</details>
 
-A few other common libraries are tools also required such as openssl 1.1+, libcurl, curl, libusb, GMP, Python 3, and zlib.
+Once you have a `*.deb` file downloaded for your version of Ubuntu, you can install it as follows.
+```bash
+sudo apt-get install -y ~/Downloads/leap*.deb
+```
+Your download path may vary.
+- If you are in an Ubuntu docker container, omit `sudo` because you run as `root` by default.
 
-**A Warning On Parallel Compilation Jobs (`-j` flag)**: When building C/C++ software often the build is performed in parallel via a command such as `make -j $(nproc)` which uses the number of CPU cores as the number of compilation jobs to perform simultaneously. However, be aware that some compilation units (.cpp files) in Leap are extremely complex and will consume nearly 4GB of memory to compile. You may need to reduce the level of parallelization depending on the amount of memory on your build host. e.g. instead of `make -j $(nproc)` run `make -j2`. Failures due to memory exhaustion will typically but not always manifest as compiler crashes.
+Finally, verify Leap was installed correctly.
+```bash
+nodeos --full-version
+```
+You should see a [semantic version](https://semver.org) string print out followed by a `git` commit hash with no errors. For example:
+```
+v3.1.2-0b64f879e3ebe2e4df09d2e62f1fc164cc1125d1
+```
 
-Generally we recommend performing what we refer to as a "pinned build" which ensures the compiler and boost version remain the same between builds of different Leap versions (Leap requires these versions remain the same otherwise its state needs to be repopulated from a portable snapshot).
+## Source Installation
+You can also build and install Leap from source.
+
+### Prerequisites
+Recent Ubuntu LTS releases are the only operating systems that we support. Other Unix derivatives such as macOS are tended to on a best-effort basis and may not be full featured. If you aren't using Ubuntu, please visit [this page](./docs/00_install/01_build-from-source/00_build-unsupported-os.md) to explore your options.
+
+Notable requirements to build are:
```

Comment:
Since this list if being changed to be exhaustive, can remove the "Notable" word. You'll listed everything.

Reply:
Done. Idk if it is exhaustive or not, but I changed it to "Requirements to build."

Reply:
It should be. The original text was trying to be both exhaustive and concise: requiring something like zlib or openssl is a nothingburger so it didn't deserve being in a list, but needing specific versions of LLVM is notable so it is brought out in to a list.

Reply:
I am not opposed to putting it back the way it was because I agree with you. However, it did seem silly to me to put some in a bulleted list and then others in prose. If they aren't important, we should just exclude them entirely IMHO.